pkg/helm/history_internal_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2021 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"helm.sh/helm/v3/pkg/release"
+)
+
+func TestSortAndTrimHistory(t *testing.T) { //nolint:funlen
```

Comment:
I'd have the following tests:
- GetHistory
  - returns_at_most_given_max_elements
  - returns_releases_with_descending_order_by_version